GestureHandler.js

```diff
@@ -748,10 +748,15 @@ const RawButton = createNativeWrapper(
 class BaseButton extends React.Component {
   static propTypes = {
     ...RawButton.propTypes,
+    exclusive: PropTypes.bool,
```

Comment:
why is it not in `RawButton` then?

Reply:
I think if someone uses RawButton then the defaults behavior is going to be different. Maybe we should defined the defaults on the native side in this case. That is when we create new native view instance it should have `exclusiveTouch` enabled by default and we don't need to have a default value specified in JS

Reply:
Well, that's fine. I changed it

Reply:
So I understand this should no longer be needed here (bc `RawButton.propTypes` already contains `exclusive`)